tuf/client/updater.py

```diff
@@ -129,6 +129,8 @@
 import copy
 import warnings
 
+from securesystemslib import exceptions as sslib_exceptions
```

Comment:
Given that the exceptions module is one of the few places in sslib where we have classes, wouldn't it be cleaner to import only the exceptions we want/need? Or do some of the conflicting exception names between sslib and tuf prevent this?

Reply:
This would probably be a good idea

Reply:
I've not done this for now... there are cases where clashes are definitely possible and a couple where they actually exist right now: the main issue is the generic "Error" exception that both projects use.

It's an idea worth exploring but I think not as part of this PR.

Reply:
Sounds reasonable, thanks for looking into it.